### ISSUE TITLE
Second Phase One color palette for all the pages - Finalize

### ISSUE THREAD

**Issue by sowmyaarajan** [alan-turing-institute/AutSPACEs#131]
### Summary
Making sure that the color palette remains the same for all the pages. The initial look for the basic MVP should be appealing to the users and shouldn't be too bright for the users. 

### Who can help?

* Anybody from the platform team

**Comment by GeorgiaHCA** [comment]
Great work @soledadli and @BrainonSilicon !
